source/renderer/app/components/widgets/Dialog.scss

```diff
@@ -53,6 +53,10 @@
     flex-shrink: 0;
     margin-top: 10px;
 
+    &:not(:empty) {
+      margin-top: 30px;
```

Comment:
this doesn't solve problem

Reply:
<img width="727" alt="image" src="https://user-images.githubusercontent.com/2539345/161541184-98bfdd97-bf03-4d8a-87cf-985bb0269168.png">
now, with expansion of bottom area scroll end position is off. It should be at the same level like the end of input field

Reply:
I fixed the scroll position. As for the change in the margin, there is no more margin bottom on the dialog content, so the scroll will appear when necessary without take in account the margin bottom that was removed.

Reply:
does it match this https://zpl.io/VkABvQg ?

Reply:
It should match now.